src/styles/utils.scss

```diff
@@ -109,8 +109,28 @@
   background-color: mat.get-color-from-palette($background, "card");
 }
 
+.mat-bg {
+  $background: map-get($vts-light-theme, "background");
+  background-color: mat.get-color-from-palette($background, "background");
+}
+.dark .mat-bg {
+  $background: map-get(map-get($vts-dark-theme, 'color'), 'background');
+  background-color: mat.get-color-from-palette($background, "background");
+}
+
 .dense-menu .mat-mdc-menu-item {
   height: 32px;
   line-height: 32px;
   min-height: 0;
 }
+
+.nav-no-scrollbar {
+  .mat-drawer-inner-container::-webkit-scrollbar {
```

Comment:
i need more context on this css hack

Reply:
yes, this is actually applied to the sidebar. when I decided to add another section, and then a route button to the sidebar, overflowing would result in a visible scroll bar which is not aesthetically pleasing

this is the sidebar before implementing the class
![image](https://github.com/vtstats/web/assets/46742676/c7301b21-63e1-43b9-bbcf-91f0b225d802)

this is the sidebar after implementing the class
![image](https://github.com/vtstats/web/assets/46742676/49d6c3bb-0ada-489e-b08d-faeeec3d50db)

Reply:
okay, i noticed that the footer in sidebar overlays the navigation part. maybe we should make the whole sidebar scrollable on smaller screen and pushing footer below the navigation part.

Reply:
so basically,

in non-small screen width, the footer will stay sticky

while the rest, the footer will come with the sidebar (which means it will be scrolled alongside the route buttons)

is that what you meant by that, or the footer must be non-sticky regardless?

Reply:
> in non-small screen width, the footer will stay sticky

> while the rest, the footer will come with the sidebar (which means it will be scrolled alongside the route buttons)

Okay, then I will stick to this plan